geometry/eolearn/tests/test_morphology.py

```diff
@@ -16,6 +16,11 @@ class TestErosion(unittest.TestCase):
     def setUpClass(cls):
         cls.classes = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
 
+    def test_erosion_value_error(self):
+        self.assertRaises(ValueError, ErosionTask, (FeatureType.MASK_TIMELESS, 'LULC', 'TEST'), disk_radius=None)
+        self.assertRaises(ValueError, ErosionTask, (FeatureType.MASK_TIMELESS, 'LULC', 'TEST'), disk_radius=0)
```

Comment:
Now I'm confused how does this test passes since I didn't understand the thing about the `disk_radius=0` case :D.

Reply:
Ignore this.